qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -161,14 +160,8 @@ def __init__(
             self._client_params.url = auth_client.current_service_urls()["services"]["runtime"]
             self._client_params.token = auth_client.current_access_token()
             self._api_client = RuntimeClient(self._client_params)
-            try:
-                self._hgps = self._initialize_hgps(auth_client)
-            except json.decoder.JSONDecodeError:
-                raise IBMApiError(
-                    "Unexpected response received from server. "
-                    "Please check if the service is in maintenance mode "
-                    "https://docs.quantum.ibm.com/announcements/service-alerts."
-                )
+            self._hgps = self._initialize_hgps(auth_client)
+
```

Comment:
For maintenance outages the whole service including auth is down so this block would not be reached.